quantities/astronomy.hpp

```diff
@@ -11,13 +11,15 @@ namespace principia {
 namespace quantities {
 
 // This namespace contains units commonly used in astronomy that are not
-// accepted for use with the SI.
+// accepted for use with the SI.  See "Résolutions de la 29e Assemblée Générale
+// de l’Union Astronomique Internationale de 2015".
```

Comment:
This is specifically B3, and I do not think that your quoted name is anything other than the name of the downloaded PDF, so
```C++
// See Résolution B3 "Sur les valeurs recommandées de constantes de conversion pour une sélection
// de propriétés solaires et planétaires" adopted at the XXIX General Assembly of the IAU in 2015.

Reply:
Done.